examples/tutorials/3d-perspective-image.py

```diff
@@ -94,55 +81,23 @@
 
 ########################################################################################
 # The ``perspective`` azimuth can be changed to set the direction that is "up"
-# in the figure.
+# in the figure. The ``contourpen`` parameter sets the pen used to draw contour lines
+# on the surface. :meth:`pygmt.Figure.colorbar` can be used to add a color bar to the
+# figure. The ``cmap`` argument does not need to be passed again. To keep the color
+# bar's alignment similar to the figure, use ``True`` as the ``perspective`` argument.
```

Comment:
See #886 for discussions and please replace "argument" with "parameter" in other instances of these two tutorials.
```suggestion
# figure. The ``cmap`` parameter does not need to be passed again. To keep the color
# bar's alignment similar to the figure, use ``True`` as the ``perspective`` parameter.
```